account_payment_order/security/ir.model.access.csv

```diff
@@ -4,3 +4,5 @@ access_account_payment_line,Full access on account.payment.line to Payment Manag
 access_bank_payment_line,Full access on bank.payment.line to Payment Manager,model_bank_payment_line,group_account_payment,1,1,1,1
 base.access_res_partner_bank_group_partner_manager,Full access on res.partner.bank to Account Payment group,base.model_res_partner_bank,group_account_payment,1,1,1,1
 base.access_res_bank_group_partner_manager,Full access on res.bank to Account Payment group,base.model_res_bank,group_account_payment,1,1,1,1
+access_account_payment_line_create,access_account_payment_line_create,model_account_payment_line_create,group_account_payment,1,1,1,1
```

Comment:
Is the button that launches this accessible only to the same group?

Reply:
Yes, it is